### ISSUE TITLE
build failure `evaluation of constant value failed`

### ISSUE THREAD

**Issue by ofzo** [denoland/rusty_v8#1263]
```toml
[toolchain]
channel = "nightly"
components = ["rustfmt", "clippy"]
```

error info: 

```shell
error[E0080]: evaluation of constant value failed
    --> /Users/roadup/.cargo/registry/src/index.crates.io-6f17d22bba15001f/v8-0.64.0/src/isolate.rs:1715:3
     |
1715 |   assert!(size_of::<TypeId>() == size_of::<u64>());
     |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the evaluated program panicked at 'assertion failed: size_of::<TypeId>() == size_of::<u64>()', /Users/roadup/.cargo/registry/src/index.crates.io-6f17d22bba15001f/v8-0.64.0/src/isolate.rs:1715:3
     |
     = note: this error originates in the macro `assert` (in Nightly builds, run with -Z macro-backtrace for more info)

   Compiling swc_ecma_ast v0.100.1
For more information about this error, try `rustc --explain E0080`.
error: could not compile `v8` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
```

How to fixed?

**Comment by mmastrac** [comment]
I believe this was fixed in #1249 and released in https://github.com/denoland/rusty_v8/releases/tag/v0.74.1

**Comment by ofzo** [comment]
> I believe this was fixed in #1249 and released in https://github.com/denoland/rusty_v8/releases/tag/v0.74.1

Yeoh，it works，thank you
